storagenode/console/consoleserver/server.go

```diff
@@ -65,8 +66,13 @@ func NewServer(logger *zap.Logger, config Config, service *console.Service, list
 
 	// handle static pages
 	if config.StaticDir != "" {
```

Comment:
The checks for which filesystem to use ends up complicating logic, since there are also other places that check whether the file exists. It's somewhat less error prone to say that only use `http.FileSystem`.

Reply:
i agree. had to fix a bug on this count already. looking for a small change here for now though

Reply:
okay, so, the change needed ist o make NewServer stop taking a Config as an argument altogether and to take an http.FileSystem instead

Reply:
Do we have to remove the config entirely? the config contains the server address, so it looks that we should remove `StaticDir` field and pass an additional parameter to the `NewServer`, isn't it?

Reply:
the config struct is fine and you can leave it alone. what i'm saying is NewServer doesn't need the config argument. i don't see where the server address is used by *Server in this file?

Reply:
Oh, I see, I didn't check the whole file, I thought that it was used somewhere else as the passed config is assigned to the `Server` struct instance.